packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -45,6 +45,9 @@ export const messages = {
   force_auth_web_login_authorize_dev_hub_text: 'SFDX: Authorize a Dev Hub',
   force_auth_web_login_authorize_org_text: 'SFDX: Authorize an Org',
 
+  parameter_directory_strict_not_available:
+    'A required folder named "%s" does not exist in this workspace.',
```

Comment:
@ruthemmanuelle - Could you help verify this text change?

Reply:
Seems ok to me, unless we can say something about why the folder is required.

Reply:
We can change to 'A required _metadata_ folder named "%s" doesn't exist in this workspace.".

Basically we need a message that tells the user why there's no directory selection when a command from the command palette is executed _and_ they're not getting any directory selection (b/c they don't have the folder created before).

Reply:
Hi @muenzpraeger . Sorry for the delay in responding to this. I just saw it today. Would this work (unless you end up changing this functionality)?

'A required metadata directory named "%s" doesn't exist in this workspace. Create the directory and then run the command again.'